bin/build-acorn.js

```diff
@@ -3,14 +3,30 @@ var stream = require("stream")
 
 var browserify = require("browserify")
 var babelify = require("babelify")
+var derequire = require("derequire")
 
 process.chdir(path.resolve(__dirname, ".."))
 
+function derequireStream() {
```

Comment:
Didn't know that existed.  I'll sort out a new pull request tomorrow if I get a chance(and nobody else has already done it)